splinterd/man/splinterd.1.md

```diff
@@ -1,81 +1,298 @@
+  service).
+
+`--advertised-endpoint` `ADVERTISED-ENDPOINT`
+: Specifies the public network endpoint for daemon-to-daemon communication
+  between Splinter nodes, if the network endpoint is not public. Use the format
+  `tcp://ip:port`. (Default: Same as the network endpoint; see
+  `-n`, `--network-endpoint`.)
+
+`--bind BIND-ENDPOINT`
+: Specifies the connection endpoint for the REST API. (Default: 127.0.0.1:8080.)
+
+`--ca-file CERT-FILE`
+: Specifies the path and file name for the trusted CA certificate.
+  (Default: `/etc/splinter/certs/ca_pem`.)
+
+  Do not use this option with the `--insecure` flag.
+
+`--cert-dir CERT-DIR`
+: Specifies the directory that contains the trusted CA certificates and
+  associated key files. (Default: `/etc/splinter/certs/`.)
+
+  This option overrides the `SPLINTER_CERT_DIR` environment variable, if set,
+  and any setting in the `splinterd` configuration file.
+
+`--client-cert CERT-FILE`
+: Specifies the path and file name for the client certificate, which is
+  used by `splinterd` when it is sending messages.
+  (Default: `/etc/splinter/certs/client.crt`.)
+
+`--client-key CLIENT-KEY`
+: Specifies the path and file name for the client key.
+  (Default: `/etc/splinter/certs/client.key`.)
+
+`-c`, `--config` `CONFIG-FILE`
+: Specifies the path and file name for a `splinterd` configuration file, which
+  is a TOML file that contains `splinterd` settings. (The file name must end
+  with a `.toml` extension.) For examples, see
+  [splinterd/packaging/splinterd.toml.example](https://github.com/Cargill/splinter/blob/master/splinterd/packaging/splinterd.toml.example)
+  and
+  [splinter/splinterd/sample_configs](https://github.com/Cargill/splinter/tree/master/splinterd/sample_configs)
+  in the `splinter` repository.
+
+  Any options on the command line will override the settings in the
+  configuration file.
+
+`--database DB-URL`
+: (Optional feature) Specifies the URL for the PostgreSQL database used for
+  Biome. (Default: 127.0.0.1:5432.) This option is required when
+  `--enable-biome` is used.
+
+  Note: This flag is available when the "database" feature is enabled.
+
+`--heartbeat SECONDS`
+: Specifies how often, in seconds, to send a heartbeat. (Default: 30 seconds.)
+  Use 0 to turn off the heartbeat.
 
+  This heartbeat is used to check the health of connections to other Splinter
+  nodes.
+
+`-n`, `--network-endpoint` `NETWORK-ENDPOINT`
+: Specifies the endpoint for daemon-to-daemon communication between Splinter
+  nodes, using the format `tcp://ip:port`. (Default: 127.0.0.1:8044.)
+
+  `--node-id NODE-ID`
+: (Required) Sets a new ID for the node. The node ID must be unique across the
+  network (for all Splinter nodes that could participate on the same circuit).
+
+`--peer PEER-URL` `[,...]`
+: Specifies one or more Splinter nodes that `splinterd` will automatically
+  connect to when it starts. The *PEER-URL* argument must specify another node's
+  network endpoint, using the format `ip:port`.
```

Comment:
technically this should now include the protocol prefix `protocol_prefix://ip:port` as well, ill add a story to fix this